### PR TITLE
10 BLB cards

### PR DESCRIPTION
**Pull request by Fulgur14**
Would Glarb work correctly with X-spells like that?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/cindering_cutthroat.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Cindering Cutthroar
```
Comment (dracontes) [inline forge-gui/res/cardsfolder/upcoming/cindering_cutthroat.txt:1]: "Cutthroa**t**" rather.

File: forge-gui/res/cardsfolder/upcoming/bello_bard_of_the_brambles.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Bello, Bard of the Brambles
+ManaCost:1 R G
+Types:Legendary Creature Raccoon Bard
+PT:3/3
+S:Mode$ Continuous | Affected$ Artifact.nonEquipment+YouCtrl+CMCGE4,Enchantment.nonAura+YouCtrl+CMCGE4 | Condition$ PlayerTurn | SetPower$ 4 | SetToughness$ 4 | AddType$ Creature,Elemental | AddKeyword$ Indestructible & Haste | AddTrigger$ TrigDamageDone | Description$ During your turn, each non-Equipment artifact and non-Aura enchantment you control with mana value 4 or greater is a 4/4 Elemental creature in addition to its other types and has indestructible, haste, and "Whenever this creature deals combat damage to a player, draw a card."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bello_bard_of_the_brambles.txt:5]: cmc

File: forge-gui/res/cardsfolder/upcoming/beza_the_bounding_spring.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Name:Beza, the Bounding Spring
+ManaCost:2 W W
+Types:Legendary Creature Elemental Elk
+PT:4/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters, create a Treasure token if an opponent controls more lands than you. You gain 4 life if an opponent has more life than you. Create two 1/1 blue Fish creature tokens if an opponent controls more creatures than you. Draw a card if an opponent has more cards in hand than you.
+SVar:TrigToken:DB$ Token | ConditionCheckSVar$ XLands |  ConditionSVarCompare$ GTYLands | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You | SubAbility$ DBGainLife
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/beza_the_bounding_spring.txt:6]: spacing

File: forge-gui/res/cardsfolder/upcoming/glarb_calamitys_augur.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Glarb, Calamity's Augur
+ManaCost:B G U
+Types:Legendary Creature Frog Wizard Noble
+PT:2/4
+K:Deathtouch
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | Affected$ Land.TopLibrary+YouCtrl,Card.TopLibrary+nonLand+cmcGE4 | AffectedZone$ Library | MayPlay$ True | Description$ You may play lands and cast spells with mana value 4 or greater from the top of your library.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/glarb_calamitys_augur.txt:7]: search for some examples with ValidAfterStack$
then it should work

**Review by tool4ever — APPROVED** [review]

**Comment by Agetian** [comment]
Strange, tried merging this PR a few times and each time it tells me "Merge already in progress, merge attempt failed", even though nothing happens and nothing is merged... Can you please try, @tool4ever ? Thanks in advance!

**Comment by Agetian** [comment]
Oh, OK, finally merged :) Not sure what was wrong before.

**Pull request merged**